src/graph/Graph.js

```diff
@@ -713,7 +716,11 @@
                     return context.highlight.edge;
                 }
                 return "1px";
-            }).transition().duration(this.transitionDuration())
+            })
+            .each(function(d){
+                d3.select(this).classed("graphEdge-highlighted",!edgeMap || edgeMap[d.id()]);
+            })
```

Comment:
Same as above this can be written as:
```
    .classed("graphEdge-highlighted", function(d) {return !edgeMap || edgeMap[d.id()];})
```
By convention I typically set the class before I set the styles (as mentally I think of it as the default style and any additional ones are applied on top) - but it is only my convention:
```
        edgeElements
            .classed("graphEdge-highlighted", function(d) {return !edgeMap || edgeMap[d.id()];})
            .style("stroke-width", function (o) {
                if (edgeMap && edgeMap[o.id()]) {
```